app/src/main/java/com/klk/dailyjournal/EditActivity.kt

```diff
@@ -0,0 +1,104 @@
+package com.klk.dailyjournal
+
+import android.content.Intent
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.EditText
+import android.widget.TextView
+import com.klk.dailyjournal.data.NoteEntity
+import com.klk.dailyjournal.data.NoteRepository
+import com.klk.dailyjournal.service.EditIdStore
+import com.klk.dailyjournal.service.MoodImageStore
+import kotlinx.android.synthetic.main.activity_edit.*
+import java.net.URI
+
+
+class EditActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_edit)
+
+
+        updateNote()
+        delete()
+    }
+
+
+    fun updateNote(){
+        val up = NoteRepository.get()
```

Comment:
here you do exactly the same as at the begging of the method delete(), I think its best if you move those lines to one method to not have duplicated code